python/test/unit/test_catalogues.py

```diff
@@ -13,11 +13,11 @@ def __init__(self):
         self.tree.append(arb.mnpos, (0, 0, 0, 10), (1, 0, 0, 10), 1)
         self.props = arb.neuron_cable_properties()
         try:
-            self.cat = arb.default_catalogue()
-            self.props.register(self.cat)
+            self.props.catalogue = arb.load_catalogue('dummy-catalogue.so')
```

Comment:
Fun fact: when building with meson on Mac OS shared libraries have the `dylib` file extension, not `so`.
Something that I have generalised in the meson branch.

Reply:
HA! We would have had that in CMake but I force-overrode that behaviour.